desktop/renderer-app/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```

Comment:
This field needs to be updated.

Reply:
cc @BlackHole1

Reply:
please provide description

Reply:
This field no need in electron render, I will remove it.